lib/src/Base/Func/HyperbolicAnisotropicEnumerateFunction.cxx

```diff
@@ -275,6 +275,18 @@ void HyperbolicAnisotropicEnumerateFunction::setUpperBound(const Indices & upper
   initialize();
 }
 
+/* The marginal enumerate function */
+EnumerateFunctionImplementation HyperbolicAnisotropicEnumerateFunction::getMarginal(const Indices & indices) const
```

Comment:
```suggestion
EnumerateFunction HyperbolicAnisotropicEnumerateFunction::getMarginal(const Indices & indices) const
```

Reply:
Thank you very much for the detailed suggestions. Fixed at 8ba0bdcdc775a35dbdf8f2987f7701ee352fa488.